### DISCUSSION TITLE
Add Nintendo Wii please

### DISCUSSION THREAD

**Discussion by Erock1314** [EmulatorJS/EmulatorJS#882] [Ideas]
please

**Comment by ethanaobrien** [comment]
See

- https://github.com/EmulatorJS/EmulatorJS/issues/187#issuecomment-1080743475
- https://github.com/EmulatorJS/EmulatorJS/issues/660#issuecomment-1707637817
- https://emulatorjs.org/docs4devs/cores#unavailable-cores-that-reasons-are-known
